Dockerfile

```diff
@@ -0,0 +1,18 @@
+#Using the official ubuntu OS as a Base Image
+FROM ubuntu:16.04
+
+USER root
+
+#Configuration
+RUN pip install -r requirements/base.txt
+RUN paver install_pages
+
+#Setting up environment variables. Change the value according to the value of the ENV_VAR
+
+ENV BASIC_AUTH_USER value	
+ENV BASIC_AUTH_PASSWORD value
+ENV USER_LOGIN_EMAIL value
+ENV USER_LOGIN_PASSWORD value
+
+#Edit the CMD command to run speciifc tests
```

Comment:
^ adding on that: it would be best if the user could shell directly into the directory containing the tests AND have the virtual environment activated (if we continue to use it)